### ISSUE TITLE
Call to mapbox Api returns "not found"

### ISSUE THREAD

**Issue by danieledep** [anvaka/mars#1]
Hi i cloned your repo and created my own token.js file, but when i open the browser the map is empty and the console throws these two errors

`Object { status: 404, url: "https://api.mapbox.com/v4/anvaka.8ctdbgc9.json?secure&access_token= ... my token ...", name: "e", message: "Not Found", stack: "" }`

`Object { status: 404, url: "https://api.mapbox.com/v4/anvaka.40t5yrsw.json?secure&access_token=... my token ...", name: "e", message: "Not Found", stack: "" }`

**Comment by anvaka** [comment]
Did you find out what happened?

**Comment by danieledep** [comment]
No, it seems these two json files are not available anymore, did you look into it?

**Comment by anvaka** [comment]
I was able to open the map of Mars - it worked! I wonder if I messed up something with mapbox permissions in mapbox studio or maybe your key doesn't have the right origin?

~~Can you see in the network tools what mapbox gives in failing responses?~~ I see you said "not found". Hmm. Let me check mapbox studio settings

**Comment by anvaka** [comment]
The map style is marked as "public" on my end. Still not sure why it gives 404.

**Comment by danieledep** [comment]
I double checked my mapbox settings as well and it is a basic token with no restrictions, and all public scopes ticked.

Maybe the Response Headers might give any tip?

> Request URL: https://api.mapbox.com/v4/anvaka.8ctdbgc9.json?secure&access_token=MAPBOX_TOKEN
Request Method: GET
Status Code: 404 Not Found (from disk cache)
Remote Address: 143.204.9.103:443
Referrer Policy: no-referrer-when-downgrade
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: Link
Cache-Control: max-age=43200,s-maxage=300
Content-Length: 23
Content-Type: application/json; charset=utf-8
Date: Mon, 25 May 2020 10:35:22 GMT
ETag: W/"17-SuRA/yvUWUo8rK6x7dKURLeBo+0"
Vary: Accept-Encoding
Via: 1.1 fba39238fd8fc98a9491893a18564594.cloudfront.net (CloudFront)
X-Amz-Cf-Id: 4qrcbgs3Hv8TofI6RmM4vxFnXcxgQBrgaKGPqsS1vAMV5qglzLxpGQ==
X-Amz-Cf-Pop: MXP64-C1
X-Cache: Error from cloudfront
X-Content-Type-Options: nosniff
X-Powered-By: Express
X-Rate-Limit-Interval: 60
X-Rate-Limit-Limit: 100000
X-Rate-Limit-Reset: 1590402982
Accept: application/json
Referer: http://0.0.0.0:8080/?theme=RED#2.64/18.79/-92.88
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
secure: 
access_token: MAPBOX_TOKEN
